cmd/juju/controller/destroy.go

```diff
@@ -232,20 +240,7 @@ func (c *destroyCommand) Run(ctx *cmd.Context) error {
 		return errors.Annotate(err, "getting controller environ")
 	}
 
-	if err := c.ConfirmationCommandBase.Run(ctx); err != nil {
-		return errors.Trace(err)
-	}
-
-	if c.ConfirmationCommandBase.NeedsConfirmation() {
-		updateStatus := newTimedStatusUpdater(ctx, api, controllerEnviron.Config().UUID(), clock.WallClock)
-		modelStatus := updateStatus(0)
-		if err := printDestroyWarning(ctx, modelStatus, controllerName, c.releaseStorage); err != nil {
-			return errors.Trace(err)
-		}
-		if err := jujucmd.UserConfirmName(controllerName, "controller", ctx); err != nil {
-			return errors.Annotate(err, "controller destruction")
-		}
-	}
+	_, _ = fmt.Fprintf(ctx.Stderr, destroySysMsg, controllerName)
```

Comment:
I would use ctx.Warningf here

If you do, you should drop `WARNING!` from destroySysMsg as ctx.Warningf handles that for you